src/device/environment.js

```diff
@@ -79,7 +79,7 @@ createNameSpace("realityEditor.device.environment");
         transformControlsSize: 1, // on remote operator, we can scale down the gizmo size for moving groundplane anchors
         localServerPort: 49369, // the port where a local vuforia-spatial-edge-server can be expected
         screenTopOffset: 0, // if there's a menubar on the top, increase this
-        maxAvatarIcons: 5, // limits the number of circular icons depicting how many avatars are currently connected
+        maxAvatarIcons: 7, // limits the number of circular icons depicting how many avatars are currently connected
```

Comment:
I wonder if it would make sense to change this based on device type? Does 7 look alright on the iPhone?

Reply:
Agreed – there's a line in the pop-up-onboarding-addon's index.js that reduces the number to 3 on mobile devices. Eventually we might want something even smarter to maximize usage of the available screen real estate, but I think that'll do for now.

        isRemoteOperator = !realityEditor.device.environment.isARMode();
        if (!isRemoteOperator) {
            ...
            env.maxAvatarIcons = 3;
        }